rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -280,28 +280,31 @@ void gs_frame::flip(draw_context_t, bool /*skip_frame*/)
 {
 	static Timer fps_t;
 
-	++m_frames;
-
-	if (fps_t.GetElapsedTimeInSec() >= 0.5)
+	if (m_fps_title || g_cfg.misc.show_fps_in_title)
```

Comment:
It is more likely to have the cfg value true than the member

Reply:
We need both. m_fps_title is always one behind. Assume cfg value goes to false just before the statement. We would never remove the FPS title part.

Reply:
That's not what I meant.
Practically the statement will return true on g_cfg.misc.show_fps_in_title more often than on m_fps_title, so it's the wrong way around.
But I guess it wouldn't matter since it should only happen once everytime the setting is switched